upgrade/upgrade-scripts/upgrade

```diff
@@ -329,16 +331,31 @@ done
 shift $((OPTIND - 1))
 
 case "$1" in
-in-place)
+deferred)
+	UPGRADE_TYPE="DEFERRED"
```

Comment:
I'm OK with this.. but I probably would prefer to put:
```
UPGRADE_TYPE="${1^^}"
```
just before the `case` statement, so that we only need to have a single line to set the `UPGRADE_TYPE` variable, and it'll "just work" if we ever add another type of upgrade..

but I'm OK with this approach too, if folks prefer this..

Reply:
I also wanted to do this but opted to move it into the `case` block since `rollback` also uses the `UPGRADE_TYPE` variable and did not want to pass that value into that function.